great3sims/psf.py

```diff
@@ -1382,10 +1382,22 @@ class DrawPSFBuilder(PSFBuilder):
     def __init__(self, obs_type, multiepoch, shear_type, draw_psf_src):
         self.obs_type = obs_type
         self.draw_psf_src = os.path.abspath(draw_psf_src)
+        filename = os.path.basename(self.draw_psf_src)
+        fin = open(self.draw_psf_src, "r")
+        #   The input file draw_psf_src is a file which contains a list of all of the available Psfs
+        #   It contains the name of the mini-library which contains the psf, the "GroupIndex" which
+        #   is the numeric index of the library, the "PsfIndex", which is the hdu index in the mini-
+        #   library, and the "PsfNumber", which is the index of the star in the PhoSim pars file
```

Comment:
Again, it's trivial, but the extra indentation in many of your comment blocks is weird.